paperqa/settings.py

```diff
@@ -253,19 +257,17 @@ class AgentSettings(BaseModel):
     # TODO: make this prompt more minimalist, instead improving tool descriptions so
     # how to use them together can be intuited, and exposing them for configuration
     agent_prompt: str = (
-        "Answer question: {question}\n\nSearch for papers, gather evidence, collect"
-        " papers cited in evidence then re-gather evidence, and answer. Gathering"
-        " evidence will do nothing if you have not done a new search or collected new"
-        " papers. If you do not have enough evidence to generate a good answer, you"
-        " can:\n- Search for more papers (preferred)\n- Collect papers cited by"
-        " previous evidence (preferred)\n- Gather more evidence using a different"
-        " phrase\nIf you search for more papers or collect new papers cited by previous"
-        " evidence, remember to gather evidence again. Once you have five or more"
-        " pieces of evidence from multiple sources, or you have tried a few times, call"
-        " {gen_answer_tool_name} tool. The {gen_answer_tool_name} tool output is"
-        " visible to the user, so you do not need to restate the answer and can simply"
-        " terminate if the answer looks sufficient. The current status of"
-        " evidence/papers/cost is {status}"
+        "Use the tools to answer the question: {question}\n\nThe {gen_answer_tool_name}"
```

Comment:
fwiw does this still align with the paper?

Reply:
The `contracrow.json` and `wikicrow.json` specify the original prompt 👌 

This one is the default moving forward, because it's more flexible to tools